fvm/meter/weighted/meter.go

```diff
@@ -36,12 +36,10 @@ var (
 		common.MemoryKindCharacter:               24,
 		common.MemoryKindMetaType:                0,
 		common.MemoryKindNumber:                  8,
-		common.MemoryKindArrayBase:               57,
-		common.MemoryKindArrayLength:             1024,
-		common.MemoryKindDictionaryBase:          33,
-		common.MemoryKindDictionarySize:          1024,
-		common.MemoryKindCompositeBase:           233,
-		common.MemoryKindCompositeSize:           1024,
+		common.MemoryKindArrayBase:               57 + 48,
+		common.MemoryKindDictionaryBase:          33 + 96,
+		common.MemoryKindCompositeBase:           233 + 96,
+		common.MemoryKindSimpleComposite:         73 + 96,
```

Comment:
Not sure, but I was under the impression we are not going to use atree map for `SimpleComposite` type.

Reply:
oh yea good point, that's my mistake.